### DISCUSSION TITLE
Does GlobaLeaks maintain active Tor connections?

### DISCUSSION THREAD

**Discussion by DannyTre** [globaleaks/globaleaks-whistleblowing-software#3363] [Q&A]
Hi all,
in our company we installed  on premise Globaleaks version 4.10.14 to handle Whistleblowing reports.
In "Network settings"-"Tor menu" --> we set:
- "NOT anonymize outgoing connections" and 
- "Make the platform reachable without Tor".

Analyzing the network traffic however, we see active connections to 3 Tor exit nodes:
- 51.81.201.207, 
- 129.13.131.140, 
- 178.254.22.21.
Is it possible that these connections are kept active by Globaleaks?
Thanks!
Danny

**Comment by evilaliv3** [comment]
Hello @DannyTre 

Thank you for your question.

Yes i confirm that any globaleaks platform maintains connenction to the Tor network in order to implement an Onion Service and enabling users to access the system fully anonymously and bypassing censorship (e.g. DNS blocks).

The IP may vary over time because Tor circuits are updated every 10 minutes selecting new nodes.

**Reply by DannyTre** [reply]
Hi Giovanni, 
thanks for your reply, I got it.
Regards
Danny
